packages/pluggableWidgets/popup-menu-web/src/__tests__/PopupMenu.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { shallow, ShallowWrapper } from "enzyme";
```

Comment:
Nice tests 👍 I notice some coversge missing, such as the "click outside" and "correct position" functionality. Perhaps you want to test this?

Reply:
coverage 😂

Reply:
Review comment on your review comment. 🤯

Reply:
Thanks, spelling police is here! 👮 😆

Reply:
Spoke with Wesley; this can be done in e2e.